### PR TITLE
chore: remove @umpirsky/country-list package to save space

### LINKED ISSUE #6421: Reduce package size due to consumption of @umpirsky/country-list
In [here](https://github.com/celo-org/celo-monorepo/blob/6148646a29e5e719df9a1b32d1580ea7e6e61be8/packages/sdk/utils/src/types.d.ts) we use  @umpirsky/country-list as a dependency and it seems to be huge:
```
du -sh @umpirsky/country-list
  96M	@umpirsky/country-list
```
We need only one 4Kb JSON file from this package, so there is potential for optimization here

### PR DESCRIPTION
**Pull request by obasilakis**
### Description

Copied what we needed from @umpirsky/country-list to a folder and removed it to reduce package size.

### Related issues

- Fixes #6421

### CODE REVIEW

File: packages/sdk/utils/data/countries/es/country.json [hunk 1]
```diff
@@ -0,0 +1 @@
...a","EH":"S\u00e1hara Occidental","WS":"Samoa","AS":"Samoa Americana","BL":"San Bartolom\u00e9","KN":"San Crist\u00f3bal y Nieves","SM":"San Marino","MF":"San Mart\u00edn","PM":"San Pedro y Miquel\u00f3n","VC":"San Vicente y las Granadinas","SH":"Santa Elena","LC":"Santa Luc\u00eda","ST":"Santo Tom\u00e9 y Pr\u00edncipe","SN":"Senegal","RS":"Serbia","SC":"Seychelles","SL":"Sierra Leona","SG":"Singapur","SX":"Sint Maarten","SY":"Siria","SO":"Somalia","LK":"Sri Lanka","ZA":"Sud\u00e1frica","SD":"Sud\u00e1n","SS":"Sud\u00e1n del Sur","SE":"Suecia","CH":"Suiza","SR":"Surinam","SJ":"Svalbard y Jan Mayen","TH":"Tailandia","TW":"Taiw\u00e1n","TZ":"Tanzania","TJ":"Tayikist\u00e1n","IO":"Territorio Brit\u00e1nico del Oc\u00e9ano \u00cdndico","TF":"Territorios Australes Franceses","PS":"Territorios Palestinos","TL":"Timor-Leste","TG":"Togo","TK":"Tokelau","TO":"Tonga","TT":"Trinidad y Tobago","TA":"Trist\u00e1n de Acu\u00f1a","TN":"T\u00fanez","TM":"Turkmenist\u00e1n","TR":"Turqu\u00eda","TV":"Tuvalu","UA":"Ucrania","UG":"Uganda","UY":"Uruguay","UZ":"Uzbekist\u00e1n","VU":"Vanuatu","VE":"Venezuela","VN":"Vietnam","WF":"Wallis y Futuna","YE":"Yemen","DJ":"Yibuti","ZM":"Zambia","ZW":"Zimbabue"}
```
Comment (jmrossy) [inline packages/sdk/utils/data/countries/es/country.json:1]: Someone from @celo-org/devx should double check but it's likely that only this JSON file is needed. All the other versions can be removed

**Review by jmrossy — COMMENTED** [review]
Thanks for this improvement. I think we can trim it even further :)

File: packages/sdk/utils/data/countries/es/country.json (hunk 1 above)
Comment (obasilakis) [reply packages/sdk/utils/data/countries/es/country.json:1]: done :)

Comment (obasilakis) [reply packages/sdk/utils/data/countries/es/country.json:1]: done :)

File: packages/sdk/utils/src/types.d.ts [hunk 2]
```diff
@@ -1,4 +0,0 @@
-declare module '@umpirsky/country-list/data/es_AR/country.json' {
```
Comment (barbaraliau) [inline packages/sdk/utils/src/types.d.ts:1]: @gastonponti do we still need this in types just from a different source?

**Review by barbaraliau — COMMENTED** [review]
Overall LGTM. Had a question on the deletion of the types file.

File: packages/sdk/utils/src/types.d.ts (hunk 2 above)
Comment (gastonponti) [reply packages/sdk/utils/src/types.d.ts:1]: We could, but the way is used, it's the same.
We were not using that as we should, and with the `require` part is assuming the object as any, which will also work with the `esData[...]` thing, without requiring that types declaration.
We can change that using imports, but we will require some changes in the Countries class to fully use the typing.

I think that due to the scope of this PR (moving files, not modifying functions/classes), it's easier to leave it this way, and then fixing/force types in another issue if we need that

**Review by barbaraliau — APPROVED** [review]
LGTM

**Review by gastonponti — APPROVED** [review]
LGTM too

**Pull request merged**